File: platforms/python/examples/scene/scene/add_to.py
from fragmentcolor import Material, Mesh, Model, Pass, Scene, Vertex

mesh = Mesh()
mesh.add_vertex(
    Vertex.pbr([0.0, 0.5, 0.0]).set(Vertex.UV0, [0.5, 1.0]),
)
model = Model(mesh, Material.pbr())

scene = Scene()
scene.add_pass(Pass("geometry"))

# Target the pass by name (or pass its index: scene.add_to(0, model)).
scene.add_to("geometry", model)

File: platforms/kotlin/examples/scene/scene/add_to.kt
import org.fragmentcolor.*

val mesh = Mesh()
mesh.addVertex( Vertex.pbr(listOf(0.0f, 0.5f, 0.0f)).set("uv0", floatArrayOf(0.5f, 1.0f)), )
val model = Model(mesh, Material.pbr())

val scene = Scene()
scene.addPass(Pass("geometry"))

// Target the pass by name (or pass its index: scene.addTo(0, model)).
scene.addTo("geometry", model)